CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[BREAKING CHANGE]** Removed color abstraction and only use palette instead
```

Comment:
Are you ready to update Kairos with that breaking change? 😱

Reply:
Not yet, I'll prepare the branch before release ;)